Jenkinsfile_k8s

```diff
@@ -1,18 +1,178 @@
+                values:
+                  - linux
+  restartPolicy: "Never"
+  containers:
+    - name: "node"
+      image: "node:16.13.1"
+      securityContext:
+        privileged: false
+      tty: true
+      command:
+      - cat
+    - name: "ruby"
+      image: "ruby:2.6.9"
+      securityContext:
+        privileged: false
+      tty: true
+      command:
+      - cat
+    - name: "blobxfer"
+      image: "mcr.microsoft.com/blobxfer:1.9.1"
+      command:
+      - cat
+      securityContext:
+        privileged: false
+      tty: true
+      '''
+    }
+  }
+
+  environment {
+    TZ = "UTC"
+    USE_LOCAL_NODE = "true"
+    USE_LOCAL_RUBY = "true"
+    HOME = "/tmp"
+  }
+
+  triggers {
+    cron("${env.BRANCH_NAME == 'master' ? 'H/30 * * * *' : ''}")
+  }
+
+  options {
+    timeout(time: 60, unit: 'MINUTES')
+    ansiColor('xterm')
+    disableConcurrentBuilds()
+    buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '5', numToKeepStr: '5')
+  }
 
-/* Only keep the 10 most recent builds. */
-def projectProperties = [
-    [$class: 'BuildDiscarderProperty',strategy: [$class: 'LogRotator', numToKeepStr: '5']],
-]
+  stages {
+    stage('NPM Install') {
+      steps {
+        container('node') {
+          sh 'npm install'
+        }
+      }
+    }
 
-def principalBranch = 'master'
+    stage('Bundle Install') {
+      steps {
+        container('ruby') {
+          // throw errors if Gemfile has been modified since Gemfile.lock
+          sh 'bundle config --global frozen 1'
+          sh 'bundle install'
+        }
+      }
+    }
 
-/* Start by building the image, and eventually pushing it if the execution is on the master branch */
-buildDockerAndPublishImage('jenkinsio', [mainBranch: principalBranch])
+    stage('Build') {
+      steps {
+        container('node') {
+          sh 'make assets'
+        }
+        container('ruby') {
+          sh 'make fetch'
+        }
+        container('ruby') {
+          sh 'make real_generate'
+        }
+      }
+    }
 
-if (env.BRANCH_NAME == principalBranch) {
-  withCredentials([string(credentialsId: 'updatecli-github-token', variable: 'UPDATECLI_GITHUB_TOKEN')]) {
+    /*
+    stage('Deploy to azure') {
+      when {
+        branch "master"
+      }
+      environment {
+        BLOBXFER_STORAGEACCOUNTKEY = credentials('BLOBXFER_STORAGEACCOUNTKEY')
+      }
+      steps {
+        container('blobxfer') {
+          sh('''
+              blobxfer upload \
+            --local-path build/_site \
+            --storage-account-key $BLOBXFER_STORAGEACCOUNTKEY \
+            --storage-account prodjenkinsio \
+            --remote-path jenkinsio \
+            --recursive \
+            --mode file \
+            --skip-on-md5-match \
+            --file-md5 \
+            --connect-timeout 30
+          ''')
+        }
+      }
+    }
+    */
 
-    updatecli(action: 'apply')
+    stage('Deploy master to netlify') {
```

Comment:
Do we need this?

Reply:
I put it in because I was thinking we could get fastly to use netlify as a origin instead of k8s. It's easy to pull out before we merge.